lxd/storage_volumes_snapshot.go

```diff
@@ -779,29 +812,29 @@ func autoCreateCustomVolumeSnapshotsTask(d *Daemon) (task.Func, task.Schedule) {
 	return f, schedule
 }
 
-func autoCreateCustomVolumeSnapshots(ctx context.Context, d *Daemon, volumes []db.StorageVolumeArgs) error {
-	// Make the snapshots
+func autoCreateCustomVolumeSnapshots(ctx context.Context, d *Daemon, volumes []db.StorageVolumeArgs) {
+	// Make the snapshots.
 	for _, v := range volumes {
-		ch := make(chan error)
+		ch := make(chan struct{})
 		go func() {
```

Comment:
@stgraber @monstermunchkin I wondered when reading this, whether it was a good idea to immediately trigger all scheduled snapshots for a member concurrently. This could potentially cause a lot of I/O (especially for dir storage pools). Perhaps we should limit the concurrency here? Perhaps on a per-pool basis?

Reply:
That's a very good point and indeed could cause big load spike. Personally, I'd be happy to do them sequentially.

Reply:
@stgraber cool, will change this now to sequential.

Reply:
@stgraber actually on closer inspection, this is already doing it sequentially, the go routine is just there to allow a select on the ctx cancellation change. I've updated the PR by adding a couple of comments to that function for clarity, but otherwise it remains unchanged.